tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs

```diff
@@ -99,5 +99,57 @@ public void VerifyEnvironmentVariables(ImageDescriptor imageDescriptor)
 
             VerifyCommonEnvironmentVariables(variables, imageDescriptor);
         }
+
+        [SkippableTheory("4.6.2", "4.7", "4.7.1", "4.7.2")]
+        [Trait("Category", "sdk")]
+        [MemberData(nameof(GetImageData))]
+        public void VerifyNuGetCli(ImageDescriptor imageDescriptor)
+        {
+            string baseBuildImage = ImageTestHelper.GetImage("sdk", imageDescriptor.Version, imageDescriptor.OsVariant);
+            string appId = $"nuget-{DateTime.Now.ToFileTime()}";
+            string command = "nuget help";
+            string output = ImageTestHelper.DockerHelper.Run(image: baseBuildImage, name: appId, command: command);
+            Assert.StartsWith("NuGet Version:", output);
```

Comment:
(Nit) The Spacing is inconsistent between test cases.  The other tests include a blank line before the validation logic.